Sources/KontextSwiftSDK/Extensions/UIApplication+Convenience.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+extension UIApplication {
+    /// Returns the currently active key window
+    var currentKeyWindow: UIWindow? {
+        UIApplication.shared.connectedScenes
+            .filter { $0.activationState == .foregroundActive }
+            .compactMap { $0 as? UIWindowScene }
+            .first?.windows
+            .filter { $0.isKeyWindow }
+            .first
+    }
+
+    /// Returns the top most presented controller
+    var topMostViewController: UIViewController? {
+        guard let keyWindow = UIApplication.shared.currentKeyWindow else {
+            return nil
+        }
+
+        var topController = keyWindow.rootViewController
+
+        while let presentedViewController = topController?.presentedViewController {
+            topController = presentedViewController
+        }
```

Comment:
Not sure if we can run into issues with TabBar or NavigationController 🤔 
But should be fine and we will offer option to overload this in future.